### PR TITLE
Save daily docket information

### PR DESCRIPTION
**Pull request by sharonwarner**
Connects #2813 

- [ ] Confirmed we save information to the backend every 30 seconds
- [ ] Confirmed that if the request errors out, we show an error and try again in 30 seconds

### CODE REVIEW

File: spec/feature/hearings_spec.rb [hunk 1]
```diff
@@ -90,6 +90,13 @@
       expect(page).to have_content("Upcoming Hearing Days")
     end
 
+    scenario "Daily docket saves to the backend" do
+      visit "/hearings/dockets/2017-01-06"
+      fill_in "1.notes", with: "This is a note about the hearing!"
```
Comment (aroltsch) [inline spec/feature/hearings_spec.rb]: @sharonwarner per our conversation, will you please add tests for the rest of the fields

File: lib/fakes/hearing_repository.rb [hunk 2]
```diff
@@ -16,8 +16,9 @@ def self.hearings_for_appeal(appeal_vacols_id)
   end
 
   def self.update_vacols_hearing!(vacols_record, hearing_info)
-    return if (hearing_info.keys.map(&:to_sym) & [:notes, :aod, :disposition, :hold_open, :transcript_requested]).empty?
-    hearing = find_by_vacols_id(vacols_record[:vacols_id])
+    return if (hearing_info.keys.map(&:to_sym) &
+        [:notes, :aod, :disposition, :add_on, :hold_open, :transcript_requested]).empty?
+    hearing = find_by_vacols_id(vacols_record[:vacols_id].to_s)
```
Comment (aroltsch) [inline lib/fakes/hearing_repository.rb]: cool!

File: spec/feature/hearings_spec.rb (hunk 1 above)
Comment (sharonwarner) [reply spec/feature/hearings_spec.rb]: Ah yeah, I totally forgot, thanks!

**Comment by sharonwarner** [comment]
Adding ticket to update dropdowns in feature test: https://github.com/department-of-veterans-affairs/caseflow/issues/3399

**Pull request merged**